astroquery/alma/core.py

```diff
@@ -20,6 +20,7 @@
 from astropy import units as u
 from astropy.time import Time
 from astropy.coordinates import SkyCoord
+import regions
```

Comment:
regions is optional dependency, so do the same try/except import the other modules using it do (mocserver and xmatch, or even here in alma)